libfreerdp/core/transport.c

```diff
@@ -1015,43 +1163,41 @@ int transport_check_fds(rdpTransport* transport)
 		return -1;
 	}
 
-	dueDate = now + transport->settings->MaxTimeInCheckLoop;
+	// dueDate = now + transport->settings->MaxTimeInCheckLoop;
```

Comment:
Must not be removed without replacement

Reply:
ok, got it, for me looks a bit awkward, but we will not use this api anyway, so i just revert this for now.

and what about 
CreateEvent
ResetEvent
SetEvent
sequence for transport->rereadEvent, i have not found any usage of this, assuming it can be safely dropped ? (GetFileDescriptors must also be changed a bit to drop it completely)
i can guess what it was intended to be used in external polling in different thread, but i can't even find sufficient api to do so, correct me if i wrong.

Reply:
about timers. for me it looks like timeout should be implemented on polling side outside of transport_check_fds, also `while (now < dueDate)` should not be in transport_check_fds, but we can leave it for future refactoring.